server/lib/nl_detection.py

```diff
@@ -0,0 +1,65 @@
+# The following classes are used for the NL Detection.
+from abc import ABC
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+
+@dataclass
+class Place:
+  """Place attributes."""
+  dcid: str
+  name: str
+
+
+@dataclass
+class SV:
+  """Statistical Variable attributes."""
+  dcid: str
+  name: str
+  place_type: str
+
+
+@dataclass
+class PlaceDetection:
+  """Various attributes of place detection."""
+  query: str
+  query_without_place: str
+  places_found: List[str]
```

Comment:
place_tokens?

Reply:
I guess tokens are not accurate here. It would be better to differentiate raw string and Place object.